### DISCUSSION TITLE
How to combine span styles?

### DISCUSSION THREAD

**Discussion by seebaermichi** [jacksleight/statamic-bard-texstyle#63] [Q&A]
Hi,

I just added the the pro bard-textstyle addon to a project of my customer and faced a big issue. If I add a few custom styles of type "span" it looks like I can't combine them. Is there a way to do so? E.g. I have two styles for the font-family and three more for different font-weight. Now when I want to set one of the font-family and in addition set a font-weight, the font-family style is removed. Probably I'm doing something wrong. How could I make sure, that it's possible to add more then one style to a span?

Many thanks in advance.

**Comment by jacksleight** [comment]
Hey, fixed in [v3.1.6](https://github.com/jacksleight/statamic-bard-texstyle/releases/tag/3.1.6).

Surprised that hasn't come up before! 😄

**Comment by seebaermichi** [comment]
Hi Jack,

wow, that was quick. Thanks a lot. 🙏
